README.md

```diff
@@ -1,5 +1,10 @@
 # rinja
 
+[![Crates.io](https://img.shields.io/crates/v/rinja?logo=rust&style=flat-square&logoColor=white "Crates.io")](https://crates.io/crates/rinja)
+[![GitHub Workflow Status](https://img.shields.io/github/actions/workflow/status/rinja-rs/rinja/rust.yml?branch=master&logo=github&style=flat-square&logoColor=white "GitHub Workflow Status")](https://github.com/rinja-rs/rinja/actions/workflows/rust.yml)
+[![Our book / manual](https://img.shields.io/readthedocs/rinja?label=book&logo=readthedocs&style=flat-square&logoColor=white "Our book / manual")](https://rinja.readthedocs.io/)
```

Comment:
I'd name it just "book" I think. "Our book" sounds a bit commercial like we're trying to sell a book we wrote or something like that haha

Reply:
Hehe, yeah, "book" might be the better title. :)